src/test_workflow/README.md

```diff
@@ -110,6 +110,17 @@ To run OpenSearch Dashboards BWC tests.
 opensearch-dashboards=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/1.3.5/4056/linux/x64/tar
 ```
 
+#### BWC tests on distribution bundle level
+
+The BWC tests running on distribution level are using the same framework from OpenSearch. The test cluster is spin up with the `latest` distribution bundle of provided version exclusively when the project is initialized with property `-PcustomDistributionDownloadType=bundle`.
```

Comment:
Should this be when gradle task is run with `customDistributionDownloadType` property?

Reply:
@gaiksaya Yes you are right. I think in our repo, I locked the BWC tests to only run on distribution level [here](https://github.com/opensearch-project/opensearch-build/blob/edffab782abf96390b3993ccc92425c63ab77884/scripts/default/bwctest.sh#L38). So basically I was thinking this part of doc for informative purpose. Do you think it's necessary to have it anyway? I have the same block in `opensearch-plugins` repo.

Reply:
Maybe attach the mentioned script then if anyone wants to reproduce.

Reply:
Just updated. Please help take a look again. Thanks.